iohk-monitoring/src/Cardano/BM/Data/BackendKind.lhs

```diff
@@ -39,7 +41,51 @@ data BackendKind =
     | MonitoringBK
     | TraceAcceptorBK
     | TraceForwarderBK
+    | UserDefinedBK Text
```

Comment:
We have `UserDefinedBK` _and_ `addExternalBackend`. It would be better if we set similar names, so `UserDefinedBK` and `addUserDefinedBackend` _or_ `ExternalBK` and `addExternalBackend`.